src/memray/_memray.pyx

```diff
@@ -1100,6 +1110,7 @@ cdef class FileReader:
 
         cdef AllocationLifetimeAggregator aggregator
         cdef _Allocation allocation
+        cdef int memory_records_seen
```

Comment:
Doesn't this need to be explicitly initialized to 0? I don't think Cython value-initializes local variables like it does instance attributes, and I can't immediately spot any place in the generated C++ code where this is getting set to 0.